.vscode/docs.code-snippets

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Just a quick comment (I just saw the files changed) but it may be worth updating our generator as well for the docs site but it isn't a blocker for this PR
https://github.com/GetJobber/atlantis/tree/master/packages/generators

Reply:
So I set up the plop generator to also generate the index file based on what's in the snippet (in a feedback branch).

What we can do for now is keep the vscode snippet around for ourselves since we'll be generating one-off index files. When we're done migrating everything in, we can delete the vscode snippet, and uncomment the lines I added to the plopfile (heh, funny name) to turn on the index file generation from there instead so we're covered for all new components. 

Great feedback! :)